### DISCUSSION TITLE
Missing data post 2.3.90 upgrade

### DISCUSSION THREAD

**Discussion by FriedrichKayak** [Security-Onion-Solutions/securityonion#6565] [Unsupported Versions] [answered]
I hope this is a dumb thing I did, and I hope it's reversible. However, post 2.3.90 upgrade, my previously collected data is missing (no alerts, for example) and I don't appear to be collecting any data since the upgrade. Full details are below and any help from the community would be much appreciated.

Host: Ubuntu 18.04.5
Type: Standalone SO manager node

Sequence of events:

1. Yesterday, found critical update pending due to Log4Shell (current version was 2.3.70).
2. Ran `sudo soup`, received error. 
3. Restarted machine using `sudo shutdown -r` (mistakenly thought this could help).
4. Waited until `sudo so-status` showed all services green.
5. Ran `sudo soup` once more, same error. Dug into log.
6. Saw output related to an error with encoding (couldn't set locale to `enUS-UTF8`).
7. Generated enUS-UTF8 locale using `locale-gen`.
8. Ran `sudo soup` to successful completion with no errors (I did not purposefully save this log output, so it is only accessible if soup did that on its own, but not sure where I'd find that). This took about 15 minutes or more. (I think this did some operating system updates also).
9. Waited until `sudo so-status` showed all services green.
10. Restarted machine using `sudo shutdown -r` (as all good Windows sysadmins do after updates, lol)
11. Waited until `sudo so-status` showed all services green.
12. Log in to console (this authentication is successful).
13. Observe no alerts (appears to be no data accessible to console, at least).
14. Observe my user is still listed in console.
15. Click Kibana, attempt log in, authentication **fails**.
16. Click TheHive, attempt log in, authentication **succeeds**.

Grid Status is also green:
Online Since: Dec 13, 2021 5:45 PM
Production EPS: 22
Consumption EPS: 45
Process Status: OK
Connection Status: OK
Raid Status: Unknown

**Comment by FriedrichKayak** [comment]
Current theory is that the EPS listed above means it's collecting, and there's just a problem with accessing the data (thus related to the problem of authenticating to Kibana/Elastic), and that's why the data isn't aggregated in the console.

**Comment by B3DTech** [comment]
Experiencing something similar myself.  Since the upgrade, I have no data in Alerts or Hunt, however Kibana is showing data.  Everything in grid is green, and so-status is all green as well originally.

I rebooted the manager node, and now I'm getting soc missing?  So I do so-soc-start and then it went green and now I'm getting data in Alerts and Hunt again.

**Reply by FriedrichKayak** [reply]
Very interesting- thank you for reply. My so-soc shows "OK" status even after the reboot. For all I know, Kibana itself may show my data if I can log into it.

**Comment by dougburks** [comment, marked as answer]
@FriedrichKayak Please try the following to see if either of these makes a difference:
- reset your SOC password per https://docs.securityonion.net/en/2.3/passwords.html#security-onion-console-soc
- add a new SOC account per https://docs.securityonion.net/en/2.3/adding-accounts.html#soc-thehive-fleet-cli

**Reply by FriedrichKayak** [reply]
@dougburks 

The password reset had no effect- but adding a new user seems to have worked. 

I can confirm collection was occurring during this period. When logged in with the new user, I can see data in Alerts and Hunt, and can authenticate to Kibana/Elastic. I will mark this as the answer, however, I do still need to repair the old account or delete it and re-add it.

**Reply by FriedrichKayak** [reply]
@dougburks 

Further update- now that the new user was created, some kind of synchronization seems to have happened because the original user is now working as designed. So, I do not need to delete and re-add the original user.
